autoendpoint/src/routers/webpush.rs

```diff
@@ -66,8 +68,38 @@ impl Router for WebPushRouter {
                 }
                 Err(error) => {
                     // We should stop sending notifications to this node for this user
+                    if error.is_timeout() {
```

Comment:
Start of additional metric fix.

I'm adding the node_id as a tag mostly because I want to see if there's some corruption that might be happening when we're pulling the record out of storage.